### PR TITLE
merge hiletgo into main

### PR DESCRIPTION
**Pull request by natashalrp**

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -70,6 +70,12 @@ public static final class DriveConstants {
     public static final int kRearRightTurningCanId = 9;
 
     public static final boolean kGyroReversed = false;
+
```
Comment (sjiyeng) [inline src/main/java/frc/robot/Constants.java:73]: Pls move to intakeconstants

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -185,8 +182,12 @@ private void configureButtonBindings() {
         m_intake));
 
     // Vision align
-    ControlContants.driverX.whileTrue(new TapeAlign(m_robotDrive,m_Vision));
-
+    ControlContants.driverX.whileTrue(new TapeAlign(
+        m_robotDrive,
+        m_Vision,
+        () -> ControlContants.driverController.getRawAxis(0),
```
Comment (sjiyeng) [inline src/main/java/frc/robot/RobotContainer.java:188]: Would probably change this to constants, similar to how it is where we set default command for drivebase

**Review by sjiyeng — CHANGES_REQUESTED** [review]
Overall looks good! Couple minor changes to do before merging in

**Comment by SuperWinner50** [comment]
Should be good

**Review by sjiyeng — APPROVED** [review]
Good!

**Comment by sjiyeng** [comment]
@akiyeng2 pls merge

**Pull request merged**